main.go

```diff
@@ -94,6 +94,8 @@ func main() {
 		Scheme:             scheme,
 		MetricsBindAddress: metricsAddr,
 		LeaderElection:     enableLeaderElection,
+		// Defaults to "" which means all namespaces
+		Namespace: os.Getenv("NAMESPACE"),
```

Comment:
@janiskemper please add to the readme this 'feature&behaviour' :) I think others will look for it too

Reply:
thank you :)